revdep/get_urls.R

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/Rscript
```

Comment:
I'd put this in `.dev` - `revdep` is used by `revdepcheck`. We shouldn't interfere with its folder structure IMO.

Reply:
oh, that was my first thought, but then I saw this folder. didn't realize it's related to `revdepcheck`